services/ai_client.py

```
@@ -0,0 +1,103 @@
+    """Thin client for Cloudflare Workers AI."""
+    
+    def __init__(self):
+        self.account_id = os.getenv("CLOUDFLARE_ACCOUNT_ID")
+        self.api_token = os.getenv("CLOUDFLARE_API_TOKEN")
+        self.model = os.getenv("CLOUDFLARE_AI_MODEL", "llama-3.1-8b-instruct")
+        self.timeout_ms = int(os.getenv("AI_TIMEOUT_MS", "6000"))
+        self.seed = int(os.getenv("AI_SEED", "42"))
+        self.default_max_tokens = int(os.getenv("AI_MAX_TOKENS", "800"))
+        
+        if not self.account_id or not self.api_token:
+            raise AIClientError("CLOUDFLARE_ACCOUNT_ID and CLOUDFLARE_API_TOKEN are required")
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)